docs/visual-basic/programming-guide/language-features/data-types/tuples.md

```diff
@@ -136,11 +136,11 @@ A method can return only a single value. Frequently, though, you'd like a method
 
 - You can use a tuple, which provides a lightweight solution to retrieving multiple return values.
 
-For example, the **TryParse** methods in .NET return a `Boolean` value that indicates whether the parsing operation succeeded. The result of the parsing operation is returned in a variable passed by reference to the method. Normally, a call to the a parsing method such as <xref:System.Int32.TryParse%2A?displayProperty=nameWithType> looks like the following:
+For example, the **TryParse** methods in .NET return a `Boolean` value that indicates whether the parsing operation succeeded. The result of the parsing operation is returned in a variable passed by reference to the method. Normally, a call to the a parsing method such as [Integer.TryParse](xref:System.Int32.TryParse) looks like the following:
```

Comment:
You'll need the `%2A` to specify the overview page, rather than a specific overload:

```suggestion
For example, the **TryParse** methods in .NET return a `Boolean` value that indicates whether the parsing operation succeeded. The result of the parsing operation is returned in a variable passed by reference to the method. Normally, a call to the a parsing method such as [Integer.TryParse](xref:System.Int32.TryParse%2A) looks like the following:
```